css/site.css

```
@@ -0,0 +1,30 @@
+img[src*='#left'] {
+  float: left;
+}
+
+img[src*='#right'] {
```

(As below) I'm not really a fan of this, we already have [these helper classes](https://kylelogue.github.io/mustard-ui/docs/alignment/index.html). Even without using these, you could also do:
```suggestion
img.align-right {
```
or `.float-right` or ...

Idem for left